vidarr-server/src/main/java/ca/on/oicr/gsi/vidarr/server/PriorityByWorkflow.java

```diff
@@ -23,7 +24,7 @@
 
 public final class PriorityByWorkflow implements ConsumableResource {
 
-  private List acceptedPriorities = Arrays.asList(1, 2, 3, 4);
+  private List<Integer> acceptedPriorities = Arrays.asList(1, 2, 3, 4);
```

Comment:
Please document which is considered highest priority because I think our understanding of this has flipped a few times